int32 object_id
string pose_id
---
srs_msgs/GraspConfiguration[] grasp_configuration
---
string fb
